src/main/resources/application.yaml

```diff
@@ -83,7 +83,7 @@ spring:
       maximum-pool-size: ${DARTS_API_DB_POOL_SIZE:50}
   jpa:
     database: postgresql
-    show-sql: false
+    show-sql: true
```

Comment:
Not sure we want to turn this on everywhere? Maybe set in a local profile?

Reply:
Agreed this was added temporarily and should have been removed - Updated